include/deal.II/algorithms/timestep_control.h

```diff
@@ -139,8 +139,11 @@ namespace Algorithms
     /**
      * Set size of the first step. This may be overwritten by the time
      * stepping strategy.
+     *
+     * @param[in] step The size of the first step, which may be overwritten by
+     *   the time stepping strategy.
      */
-    void start_step (double);
+    void start_step (double step);
```

Comment:
The pull request is reasonable and the information missing indeed. In addition to the change recommended by @bangerth may I ask that you change the title? The main deal.II archive cannot afford any pull requests for the purpose of practice!